src/routes/api/email/+server.ts

```diff
@@ -1,29 +1,34 @@
 import mailchimp from '@mailchimp/mailchimp_transactional';
 import { env } from '$env/dynamic/private';
-import dynamics from '$lib/dynamics';
-import type { MessageRecipient } from '@mailchimp/mailchimp_transactional';
 
 const apiKey = env.MAILCHIMP_API_KEY;
 const client = mailchimp(apiKey ?? '');
 
 export async function POST({ request }) {
 	try {
-		const { email, results } = await request.json();
+		const { email, results_string } = await request.json();
 		// todo: include email validation
-		const features = Object.keys(results);
 
 		const message = {
-			to: [{ email, type: 'to' } as MessageRecipient],
-			from_email: 'info@allwecansave.earth',
-			subject: '8 Dynamics - Your Results',
-			html: `<h1>Your Results: </h1>
-				<div>
-				${dynamics.map((d, i) => `<span>${d.short}: ${results[features[i]]}</span>`)}
-				</div>
-			`
+			template_name: 'eight-dynamics',
+			template_content: [
+				{
+					name: 'results_link',
+					content: `https://dreamy-lokum-0293b5.netlify.app/results/${results_string}`
```

Comment:
Looking great! Want to go ahead and add an environment variable for the URL now?

Since it looks like they want the link to be embedded in a paragraph, we may want to update this to be something like:

```
Thank you for spending some time with our 8 Dynamics of Climate Engagement! Your results are linked <a ... >here</a>.
```

We can then probably update our `div` in Mandrill to be a `p` tag for consistency with the rest of the email.

Reply:
yep makes sense, updating!

Reply:
any pref for env var name?

Reply:
`BASE_URL` would be great, tyty!